setup.py

```diff
@@ -24,6 +24,7 @@
         'beautifulsoup4>=4.6.0',
         'requests>=2.18.4',
         'Click>=6.7',
+        'selenium==3.141.0',
```

Comment:
this f*cking line was responsible for giving me all the build failed reports.

Reply:
This should be an optional dependency. We dont want to break android users.

Reply:
Ho, ok, will do.